spiffworkflow-frontend/src/rjsf/custom_widgets/NumericRangeField/NumericRangeField.tsx

```diff
@@ -80,11 +96,38 @@ export default function NumericRangeField({
   const onChangeLocal = (nameToChange: any, event: any) => {
     event.preventDefault();
     const numberValue = parseNumberString(event.target.value);
+    if (numberValue === null || (numberValue === 0 && required)) {
+      if (nameToChange === 'min') {
+        onChange({
+          ...(formData || {}),
+          min: null,
+        });
+      }
+      if (nameToChange === 'max') {
+        onChange({
+          ...(formData || {}),
+          max: null,
+        });
+      }
+      return;
+    }
     if (numberValue > maxNumber) {
       return;
     }
+    if (nameToChange === 'min') {
+      min = numberValue;
+      setMinValue(formatNumberString(numberValue.toString()));
+    }
+    if (nameToChange === 'max') {
+      max = numberValue;
+      setMaxValue(formatNumberString(numberValue.toString()));
+    }
     if (!disabled && !readonly) {
       if (nameToChange === 'min' && numberValue > max) {
+        min = numberValue;
+        setMinValue(numberValue.toString());
```

Comment:
i think we already called `setMinValue` above in this case on line 119. one place it has `formatNumberString` and one place it doesn't, but i'm not sure which is correct.